### PR TITLE
Introduce "showbans" command.

### PR DESCRIPTION
**Pull request by Drake103**
Implemented "showbans" command to show a list of banned users.

If this pull request is unneccesary, please reject it. It is fair.

### CODE REVIEW

**Comment by uwx** [comment]
This isn't very useful at the moment, but once temp bans are added it would be nice to show time left until user is unbanned.

**Comment by Drake103** [comment]
This list would be useful to unban users (`unban #2`).

**Comment by NaamloosDT** [comment]
Seems interesting, however I want to request a couple of changes:

1. Could you use interactivity (pagination) to list bans instead? If a guild has a lot of bans this could cause some issues character limits
2. Could you rename the command to `ListBans`?

File: ModCore/Extensions.cs [hunk 1]
```diff
@@ -42,5 +42,10 @@ public static GuildSettings GetGuildSettings(this DiscordGuild gld, DatabaseCont
 
             await db.SaveChangesAsync();
         }
+
+        public static string ToStringWithDiscriminator(this DiscordUser user)
```
Comment (knightking100) [inline ModCore/Extensions.cs]: I think this method should be named `ToDiscordTag`

Comment (Drake103) [reply ModCore/Extensions.cs]: Thank you! I have renamed that method.

**Comment by Drake103** [comment]
I have made similiar pagination like in other methods, but purely on the bot side because of api limit. And changed command name to `listbans` (`lb`).

**Comment by uwx** [comment]
nothing's showed up, so i'll merge

**Pull request merged**